### ISSUE TITLE
When I insert a ( inside a string followed by/inside a function/method call, shouldn't show tooltips

### ISSUE THREAD

**Issue by CodeCrazy-ywt** [microsoft/pylance-release#1677]
## Environment data

-   Language Server version: v2021.8.1
-   OS and version: windows 10 20H2
-   Python version (& distribution if applicable, e.g. Anaconda): 3.9.6

## Expected behaviour

Not to show tooltips when I input a ( inside a string with a method call

## Actual behaviour

Showing tooltips when I input a ( inside a string with a method call

## Logs

```
Background analysis message: setFileOpened
Background analysis message: markFilesDirty
[FG] parsing: c:\Users\Admin\ywt\2.py (0ms)
[FG] binding: c:\Users\Admin\ywt\2.py (0ms)
[FG] parsing: c:\Users\Admin\.vscode\extensions\ms-python.vscode-pylance-2021.8.1\dist\typeshed-fallback\stdlib\typing.pyi [fs read 1ms] (14ms)
[FG] binding: c:\Users\Admin\.vscode\extensions\ms-python.vscode-pylance-2021.8.1\dist\typeshed-fallback\stdlib\typing.pyi (2ms)
[FG] parsing: c:\Users\Admin\.vscode\extensions\ms-python.vscode-pylance-2021.8.1\dist\typeshed-fallback\stdlib\_typeshed\__init__.pyi [fs read 0ms] (7ms)
[FG] binding: c:\Users\Admin\.vscode\extensions\ms-python.vscode-pylance-2021.8.1\dist\typeshed-fallback\stdlib\_typeshed\__init__.pyi (1ms)
[FG] parsing: c:\Users\Admin\.vscode\extensions\ms-python.vscode-pylance-2021.8.1\dist\typeshed-fallback\stdlib\typing_extensions.pyi [fs read 0ms] (1ms)
[FG] binding: c:\Users\Admin\.vscode\extensions\ms-python.vscode-pylance-2021.8.1\dist\typeshed-fallback\stdlib\typing_extensions.pyi (1ms)
[FG] parsing: C:\Users\Admin\AppData\Local\Temp\pyright-9772-jAesaon5UXkD\builtins-9772-aG0AZQf6dxen-.py [fs read 9ms] (47ms)
[FG] binding: C:\Users\Admin\AppData\Local\Temp\pyright-9772-jAesaon5UXkD\builtins-9772-aG0AZQf6dxen-.py (13ms)
Background analysis message: analyze
[BG(1)] analyzing: c:\Users\Admin\ywt\2.py ...
[BG(1)]   parsing: c:\Users\Admin\ywt\2.py (0ms)
[BG(1)]   binding: c:\Users\Admin\ywt\2.py (0ms)
[BG(1)]   checking: c:\Users\Admin\ywt\2.py (3ms)
[BG(1)] analyzing: c:\Users\Admin\ywt\2.py (3ms)
Background analysis message: getDiagnosticsForRange
Background analysis message: getSemanticTokens delta
[BG(1)] getSemanticTokens delta previousResultId:1629034191279 at c:\Users\Admin\ywt\2.py (0ms)
Background analysis message: resumeAnalysis
Background analysis message: getDiagnosticsForRange
Background analysis message: markFilesDirty
Background analysis message: analyze
```

## Code Snippet / Additional information

```python
'string'.capitalize()
# 1. You can replace "string" with anything, and replace "capitalize" with any other method of str.
# 2. Insert a ( into anywhere inside the string, and see what appears.
```
![屏幕截图 2021-08-15 212637](https://user-images.githubusercontent.com/62426149/129480319-a7abbd91-ac47-470d-8a33-d8279d0c1281.png)

**Comment by jakebailey** [comment]
There's an edge case or two still (nested in an fstring), but this should be fixed in the next release.

**Comment by jakebailey** [comment]
This issue has been fixed in version 2021.9.3, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#202193-22-september-2021
